conda/environments/all_cuda-118_arch-x86_64.yaml

```
@@ -13,7 +13,7 @@ dependencies:
 - cudatoolkit
 - cudf==24.10.*,>=0.0.0a0
 - cuml==24.10.*,>=0.0.0a0
-- cupy>=12.0.0
+- cupy>=13.2.0
```

```suggestion
- cupy>=13.2.0
- llvmlite>=0.43.0
```
Let's see if this gives us information (if my llvmlite guess was right)...